docs/reference/index.rst

```diff
@@ -5,13 +5,13 @@ This contains details of the Python API as well as a reference to the
 command line interface.
 
 .. note::
-   We have reproduced API documentation from the `GUFE`_ package here for convenience.
-   The GUFE package serves as a foundation layer for openfe, providing abstract base classes and object models, and so might be more useful for developers.
+   We have reproduced API documentation from the `gufe`_ package here for convenience.
```

Comment:
Probably excessive but we can use a macro to not have to copy and paste this note in two places

Reply:
once we're >2 then I'll bother with a macro?

Reply:
That sounds like a good metric, see:
https://xkcd.com/1319/
https://xkcd.com/1205/